clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonReservationReport.groovy

```diff
@@ -112,4 +139,69 @@ class AmazonReservationReport implements ReservationReport {
       return (reservedVpc.intValue() - usedVpc.intValue())
     }
   }
+
+  static class Borrow {
```

Comment:
Agree - allocation seems a better choice to me, since regional RIs are just a financial concept as opposed to true AZ capacity guarantees. The way I think of it, Spinnaker is allocating these reservations to the instance types, rather than an external system performing a borrow on an offering.